content/venue/contents.lr

```diff
@@ -18,20 +18,15 @@ _template: page.html
 ---
 body:
 
+## Medellín
+
 <br />
-<br />
-<br />
-<br />
-<br />
-<br />
-<br />
-<br />
-<br />
-<br />
-<br />
-<br />
-<br />
-<br />
+
+Medellín is a city to experience with all your senses, as there are many reasons that make it a top tourist destination in South America. Known as _"La ciudad de la Eterna Primavera"_ (the City of Eternal Spring), the city enjoys throughout the year an sprint-like climate and temperature, as well as being a warm and rich cultural center.
+
+The city is also the hometown of many world-renowned artists, painters and sculptures, including Fernando Botero, whose works embellish many parks and squares around the city.
```

Comment:
remove "the": "is also hometown of many"...

Reply:
How about adding this: Medellín is often referred to as the “Cultural Center of Colombia”. There are almost 40 museums (e.g. Museum of Modern Art and an interactive museum called Parque Explora”) in the city, along with multiple theatres and even a planetarium and the largest freshwater aquarium in South America (taken from http://blog.hallocasa.com/20-insights-about-launching-a-company-in-medellin-colombia/)